notebooks/llm-question-answering/llm-question-answering.ipynb

```diff
@@ -9,14 +9,14 @@
     "\n",
```

Comment:
<p>I think need to add small description that parameters for updating generation config can be put directly into generate method without direct manipulation with generate_config object (otherwise description and code in cell bellow are not aligned) and possibly provide link on docs with supported parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2062/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I added description, but I don't find docs with all supported parameters